_data/toc.yaml

```diff
@@ -953,14 +953,12 @@ reference:
     path: /engine/reference/commandline/dockerd/
 - sectiontitle: Application Programming Interfaces (APIs)
   section:
-  - sectiontitle: Docker Engine API
+  - sectiontitle: Docker Engine SDK and API
     section:
-    - path: /engine/api/
+    - path: /develop/sdk/
```

Comment:
I prefer `/engine/api/`, and think we should consider splitting the topics into two pages.

Reason I think `/engine/api/` is better, is that the URL navigation is more logical

- https://docs.docker.com/engine/api/ -> General overview of the API, and list of API versions
- https://docs.docker.com/engine/api/v1.40/ -> documentation about a specific API version

With the above, I'd opt for removing the "sdk" sections from the page, and adding a separate page  on which we describe the SDKs. I see there's already a `/engine/api/sdks` below; we can reuse that location for this.

There's also some "enterprise API mismatch" parts mentioned that are either outdated or should be removed (I can help with that)

Reply:
`/engine/api` doesn't have an overview topic atm. It is currently redirected to `/develop/sdk`. I can create a new index file at  `/engine/api`  with changes to Engine API. 

Also, `/engine/api/sdks` doesn't exist; this is also being redirected to `/develop/sdk`. We can move topics from `/develop/sdks` to `/engine/api/sdk`/.